admin/proc-using-azure-auth-provider.adoc

```diff
@@ -0,0 +1,141 @@
+[id='proc-using-azure-auth-provider_{context}']
+= Using Microsoft Azure as an authentication provider in {product}
+
+The `core-plugin-api` package in {product-short} comes integrated with Microsoft Azure authentication provider, authenticating signing in using Azure OAuth.
+
+.Prerequisites
+* You have deployed {product-short} on AKS. For more information, see xref:proc-rhdh-deploy-aks_admin-rhdh[].
+
+.Procedure
+
+. In the Azure portal, navigate to the Microsoft Enterprise ID service and select *App Registrations*.
+. Click *New Registration* and complete the form following the provided instructions.
+. Once the application is created, note down the following:
```

Comment:
Is this an Azure UI or a RHDH UI? If the user must do this procedure on the Azure UI, then link to it here rather than embed in our docs. Similar to including upstream docs in RHDH docs, duplicating third-party procedures in our docs can cause major maintenance, support, and legal issues.

Reply:
Moved it to prereq